chapters/synchronous.tex

```diff
@@ -1103,59 +1103,60 @@ \section{Clocked Discretized Partition}\label{continuous-time-equations-in-clock
 With the language elements defined in this section, continuous-time equations can be used in clocked partitions.
 Hereby, the continuous-time equations are solved with the defined integration method between clock ticks.
 
-From the view of the continuous-time partition, the clock ticks are not interpreted as events, but as step-sizes of the integrator that the integrator must exactly hit.
+Such a partition is called a discretized partition, and the clock ticks are not interpreted as events, but as step-sizes of the integrator that the integrator must exactly hit.
 Hence, no event handling is triggered at clock ticks (provided an explicit event is not triggered from the model at this time instant).
 
 \begin{nonnormative}
 The interpretation of the clock ticks is the same assumption as for manually discretized controllers, such as the z-transform.
 \end{nonnormative}
 
 \begin{nonnormative}
-It is not defined how to handle events that are triggered while solving the continuous-time partition.
+It is not defined how to handle events that are triggered while solving a discretized partition.
 For example, a tool could handle events in the same way as for a usual simulation -- but only check them at the time associated with clock-ticks.
 
 Alternatively, relations might be interpreted literally, so that events are no longer triggered (in order that the time for an integration step is always the same, as needed for hard real-time requirements).
 However, even if relations do not generate events, when-clauses and operators \lstinline!edge! and \lstinline!change! should behave as normal.
 \end{nonnormative}
 
-From the view of the clocked partition, the continuous-time partition is discretized and the discretized continuous-time variables have only a value at a clock tick.
-Therefore, such a partition is handled in the same way as any other clocked partition.
-Especially, operators such as \lstinline!sample!, \lstinline!hold!, \lstinline!subSample! must be used to communicate signals of the discretized continuous-time partition with other partitions.
-Hereby, a discretized continuous-time partition is seen as a clocked partition.
+From the view of other partitions, the discretized continuous-time variables have only a value at a clock tick (internally it may be more complicated, see \cref{solver-methods}).
```

Comment:
Would read more naturally to me:
```suggestion
From the view of other partitions, the discretized continuous-time variables only have values at clock ticks (internally it may be more complicated, see \cref{solver-methods}).
```